### PR TITLE
Lobanoff maksim branch

### PR DESCRIPTION
**Pull request by LobanoffMaksim**

### CODE REVIEW

**Comment by umu-art** [comment]
Пропущено .detach().cpu()
